style.css

```diff
@@ -159,6 +159,62 @@ header {
   border-bottom: 1px solid #def1e6;
 }
 
+/* Contact Section */
+#contact-section {
+  background-color: #6070ff;
+  padding: 100px 20px 20px 20px;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+#contact-section h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+#contact-section p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+}
+
+#contact-form {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+#contact-form input,
+#contact-form textarea {
+  width: 100%;
+  padding: 15px;
+  border-radius: 10px;
+  border: none;
+}
+
+#contact-form button {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 15px;
```

Comment:
- [ ] You may also consider adding some styles on the `get in touch` button state `hover`, `active`, and `disable` as shown on the template.
![image](https://user-images.githubusercontent.com/97100091/212927054-618e5638-23d2-4eb6-a56a-bc18a61df705.png)

Reply:
I followed the Figma design template. You're using a different width and height from the one I used during the design. That's why you're getting different results.

![my_des](https://user-images.githubusercontent.com/56425107/212934908-7f3884e3-65ea-41e6-b071-d299789be64f.PNG)
My Design

![fig_design](https://user-images.githubusercontent.com/56425107/212934969-1e99a4ae-17f2-4db4-a31d-c6a9ca1f8b51.PNG)
Figma Design

Reply:
> I followed the Figma design template. You're using a different width and height from the one I used during the design. That's why you're getting different results.
> 
> ![my_des](https://user-images.githubusercontent.com/56425107/212934908-7f3884e3-65ea-41e6-b071-d299789be64f.PNG) My Design
> 
> ![fig_design](https://user-images.githubusercontent.com/56425107/212934969-1e99a4ae-17f2-4db4-a31d-c6a9ca1f8b51.PNG) Figma Design

@preciousbetine It is excellent following the size provided by the browser. because as long as I know the mobile version reaches up to `320px` width. For more accessibility for your page I think it's great to make sure the page display well on that size as well. But if you want I would leave it at `375px` though